### ISSUE TITLE
CK12

### ISSUE THREAD

**Issue by Popolechien** [openzim/zim-requests#277]
- Website URL: https://www.ck12.org/
- License: **CC-by-NC
- Desired ZIM Title: CK12
- Desired ZIM Description: the smarter way to learn
- Desired ZIM Icon : https://www.ck12.org/media/common/images/logo_ck12.svg
- Language (ISO 639-3): **eng**
- Is this a MediaWiki?:  no

Might be worth considering breaking it into subsets (Flexbooks, Plix, etc.)

**Comment by RavanJAltaie** [comment]
@Popolechien shall I start a recipe for this URL, or you want us to discuss splitting the content to several recipes and files?

**Comment by Popolechien** [comment]
@RavanJAltaie Let's keep it as a unique zim.

**Comment by RavanJAltaie** [comment]
Recipe created
https://farm.openzim.org/recipes/CK12_en_all
I'll update the library link once ready

**Comment by RavanJAltaie** [comment]
The recipe succeeded, but the interactive content inside the file is not opening (Loading only).
I'm investigating, changing some parameters (Scope type) to troubleshoot.
https://dev.library.kiwix.org/viewer#flexbooks_en_all_2023-11

**Comment by RavanJAltaie** [comment]
@benoit74 Could you please have a look at this?

**Comment by benoit74** [comment]
Last 3 runs have failed, I requested again the recipe to have a fresh try / ZIM, current ZIM is dated from Nov 2023, it is way too old now that we've released Zimit 2.

Note that: 
- due to upstream bug https://github.com/openzim/warc2zim/issues/325, Youtube videos will not work, but at least it should give a good overview of where we are currently with this request
- ZIM metadata does not look very correct: ZIM name does not match our convention ; title is very short ; description does not help at all to know what the ZIM is about.

**Comment by benoit74** [comment]
Task produced a ZIM in 3 minutes ... which is obviously plain wrong ^^

Question: isn't this recipe a duplicate of https://farm.openzim.org/recipes/flexbooks_en_all?

**Comment by benoit74** [comment]
I've investigated content of last ZIM produced in 3 minutes and it's really empty: https://dev.library.kiwix.org/viewer#ck12_2024-06/www.ck12.org/student/

I've opened an upstream bug.

**Comment by benoit74** [comment]
What do we really want to ZIM on www.ck12.org ? It looks like everything is on flexbook.ck12.org (handled in https://github.com/openzim/zim-requests/issues/278), where ck12 is just the "portal".

**Comment by Popolechien** [comment]
Yeah, and looks like ck12 changed quite a bit from when I opened the issue 5 years ago.
